### DISCUSSION TITLE
is Native look and feel important?

### DISCUSSION THREAD

**Discussion by suesitran** [suesitran/interactive_diary#49] [Polls]
This question is applicable to the use of Material design on Android apps, and Cupertino design on iOS app. 

What are the main differences of using Material design vs Cupertino design?
What are the differences between

**Comment by hieuwu** [comment]
This is not important anymore. There are a lots of inconsistent "look and feel" points between Android and iOS by default, it is very hard to cover everything. In terms of testing, it is more painful to point out which platform is right when some strange behaviors occurs. In my opinion, with the born of cross platform development these days, we are able to keep the look and feel of the app as consistent as possible across the platforms so we should do this instead of trying to adapt to each platform then try to keep the user behavior the same.

**Comment by suesitran** [comment]
decision: remove adaptive app, and use adaptive route and dialog
